packages/arco-lib/package.json

```diff
@@ -36,11 +36,13 @@
     "eslint-plugin-react-hooks": "^4.3.0",
     "lodash-es": "^4.17.21",
     "react": "^17.0.2",
-    "react-dom": "^17.0.2"
+    "react-dom": "^17.0.2",
+    "react-resizable": "^3.0.4"
```

Comment:
Why use react-resizable in table and use re-resizeable in editor?

Reply:
```js
// react-resizable
<Resizable handle={<span />}>
    <div id="container"></div>
</Resizable>

// render as 
<div id="container" class="react-resizable">
    <div class="handle"></div>
</div>
```

```js
// re-resizable
<Resizable>
    <div id="container"></div>
</Resizable>

// render as 
<div>
    <div id="container"></div>
</div>
```
react-resizable does not change the container, which is more suitable for scenarios like table where you need to return `th` directly. re-resizable can also be implemented, but you need to specify the return `th` manually and add the corresponding styles and child elements through the props returned by arco.      
react-resizable is more complex to use and will cause the child component to be re-rendered if it is not wrapped in memo. Whereas react-resizable simply adds a div wrap and changes its width, which is simple and does not cause the child to be re-rendered.
Also react-resizable has a mis-touch in the editor, haven't checked why yet